src/Application/Controller/XMLRPC/Handler.php

```diff
@@ -254,6 +254,28 @@ public function getTicketInfo($ticket_id)
 			
 			return $this->_getTicketInfo($ticket);
 		}
+
+		/**
+		 * Reset given ticket by ticket id to initial state.
+		 * Also reset the childs of a meta ticket and
+		 * encoding tickets which depend on the given ticket.
+		 *
+		 * @param integer $ticket_id ticket identifier
+		 * @throws Exception
+		 */
+		public function resetTicket($ticket_id)
+		{
+			$ticket = Ticket::find(['id' => $ticket_id]);
+			if (!$ticket) {
+				throw new Exception(__FUNCTION__ . ': ticket not found', 201);
+			}
+
+			if (!in_array($ticket['project_id'], $this->_assignedProjects)) {
+				throw new Exception(__FUNCTION__ . ': ticket in project not assigned to worker group', 202);
+			}
+
+			$ticket->reset($this->worker['id']);
```

Comment:
The function should return something, at least a boolean.
I know that "no tickets were reset since all matching tickets are already in their initial state" is a valid result of this method. So maybe just return "true" for the time being until we need something more complex.